packages/destination-actions/src/destinations/liveramp-audiences/operations.ts

```diff
@@ -36,12 +27,10 @@ async function processData(request: RequestClient, settings: Settings, payloads:
   }
 
   // STRATCONN-2584: verify multiple emails are handled
-  const filename = `${payloads[0].audience_name}_PII_${payloads[0].received_at}.csv`
-  const fileContent = rows.join('\n')
+  const filename = payloads[0].filename
```

Comment:
Consider using safe access operator.
```suggestion
  const filename = payloads[0]?.filename
```

Reply:
Think the type throw is adequate exception for case scenario.